packages/dart_frog_auth/README.md

```diff
@@ -0,0 +1,170 @@
+[![Dart Frog Logo][logo_white]][dart_frog_link_dark]
+[![Dart Frog Logo][logo_black]][dart_frog_link_light]
+
+[![ci][ci_badge]][ci_link]
+[![coverage][coverage_badge]][ci_link]
+[![pub package][pub_badge]][pub_link]
+[![style: very good analysis][very_good_analysis_badge]][very_good_analysis_link]
+[![License: MIT][license_badge]][license_link]
+
+Header Authorization support for [Dart Frog][dart_frog_link].
+
+Developed with 💙 by [Very Good Ventures][very_good_ventures_link] 🦄
+
+## About
+
+There are many different approaches when tackling authentication in a backend, `dart_frog_auth`
+focus on providing the foundations where more complex authentication methods can be built on top
+of.
+
+The authentication methods provided in `dart_frog_auth` are based on `Authorization` specification,
+which consist in a framework defined by [`General HTTP`][general_http], here you will find support
+for `Basic` and `Bearer` authentications, which are common methods used around the world.
+
+## Basic Authentication
+
+Like its name infers, it is a basic authentication method, that consists on the client sending
+the user's credentials in the `Authorization` header. The credentials should be sent concatenated by a
+colon and encoded in a base64 string, the encoded credentials are then set in the header as
+follows:
+
+```
+Authorization: Basic TOKEN
+```
+
+Due to the credentials being sent encoded and not encrypted, this authentication can be considered
+to have lessen security levels, specially with used out of a HTTPS/TLS context.
+
+## Bearer Authentication
+
+Similar in its format to the Basic authentication, but with the difference that instead of
+sending the users credentials on the header, a token is sent instead.
```

Comment:
```suggestion
Similarly to the basic authentication scheme, the bearer authentication scheme sends the users credentials on the header. Whereas a single token is sent instead of a username and password.
```